server/middlewares/index.js

```diff
@@ -4,6 +4,7 @@ import checkForErrors from './checkForErrors';
 import AuthorValidator from './AuthorValidator';
 import authorizeUser from './authorizeUser';
 import BookValidator from './BookValidator';
+import FavoriteBookValidator from './FavoriteBookValidator';
```

Comment:
instead of creating another file for FavouriteBookValidator, won't it be better for the validator to be a method in the BookValidator class since it's still related to books?

Reply:
Okay, I will change this.

Reply:
done